Answers.md

```diff
@@ -1 +1,16 @@
+1. A forEach loop returns the original array with the adjusted  values, whereas a .map function 
```

Comment:
right line of thought, but `forEach` doesn't return anything unless you tell it it